test/AllInOne.cs

```diff
@@ -799,7 +801,7 @@ async void Test()
             s = $@"\{p.Name}
                                    ""\";
             s = $"Color [ R={func(b: 3):#0.##}, G={G:#0.##}, B={B:#0.##}, A={A:#0.##} ]";
-            
+
```

Comment:
It's ugly, but it's part of the test. Trailing spaces are expected to be removed by Prettier. But I realize it should be done only once in the file, and with an explicit comment.